arangod/Agency/Store.cpp

```diff
@@ -692,7 +691,51 @@ bool Store::applies(arangodb::velocypack::Slice const& transaction) {
     Slice value = transaction.get(key);
 
     if (value.isObject() && value.hasKey("op")) {
-      _node.hasAsWritableNode(abskeys.at(i)).first.applieOp(value);
+      if (value.get("op").isEqualString("delete") ||
+          value.get("op").isEqualString("replace") ||
+          value.get("op").isEqualString("erase")) {
+        if (!_node.has(abskeys.at(i))) {
+          continue;
+        }
+      }
+      auto uri = Node::normalize(abskeys.at(i));
```

Comment:
I think we should also normalize the uri used to search for callbacks. This should be `Store.cpp:295`.

Reply:
It is the same as the normalization